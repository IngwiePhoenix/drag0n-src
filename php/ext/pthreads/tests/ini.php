<?php
ini_set("include_path", ":/var/lib/other");
class Test extends Thread {
	public function run(){
		printf("%s: %s\n", __METHOD__, ini_get("include_path"));
	}
}
$test = new Test();
$test->start();
?>
