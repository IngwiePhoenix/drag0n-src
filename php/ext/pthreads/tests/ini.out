Warning: Zend OPcache can't be temporary enabled (it may be only disabled till the end of request) in Unknown on line 0
Test::run: :/var/lib/other